bin/resources/GameIndex.yaml

```diff
@@ -1273,6 +1279,11 @@ SCED-50989:
 SCED-51016:
   name: "Official PlayStation 2 Magazine Demo 22/23"
   region: "PAL-M5"
+SCED-51075:
+  name: "Ratchet & Clank [Regular Demo]"
+  region: "PAL-M5"
+  gameFixes:
+    - VUKickstartHack # Fixes Character SPS.
```

Comment:
Same situation here regarding the timing hack.

Reply:
Thanks  , taken care of :+1: